e2e/process-services-cloud/people-group-cloud-component.e2e.ts

```diff
@@ -170,6 +181,127 @@ describe('People Groups Cloud Component', () => {
             groupCloudComponentPage.checkSelectedGroup(`${groupNoRole.name}`);
         });
 
+        it('[C305033] Should fetch the preselect users based on the Validate flag set to True or False', () => {
```

Comment:
Can be break this test in smaller test case? It's hard to follow the test. I would break it by component's properties: mode, validate, etc.